CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -183,6 +185,121 @@ static void __CFTimeZoneGetOffset(CFStringRef timezone, int32_t *offset) {
 
     RegCloseKey(hKey);
 }
+#elif TARGET_OS_ANDROID
+typedef Boolean (*__CFAndroidTimeZoneListEnumerateCallback)(char name[40], int32_t start, int32_t length, FILE *fp, void *context);
```

Comment:
Why the limit of the name?  If you want a nullability attribute instead, just spell it as `_Nonnull`.  Also, the name should be `const` no?

Reply:
The name can be `const` indeed, it isn’t in the original struct from Bionic, so I didn’t use `const`. I can change it.

The reason for the 40 is because that's the maximum length of the name in the original struct. There's very little documentation about if there will always be a final zero or not, so I preferred to deal with the length.

Reply:
Please add a comment explaining that.